sdk/src/main/res/layout/hcaptcha_fragment.xml

```diff
@@ -8,36 +8,43 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent" />
 
-    <LinearLayout
+    <RelativeLayout
         android:id="@+id/loadingContainer"
-        android:orientation="vertical"
-        android:gravity="center"
```

Comment:
Why this layout change?

Looks like it is not necessary.
![Uploading image.png…]()

Reply:
@DSergiu we need an extra layer to prevent clicks delivered to WebView 

> 2. Also, clicking on the overlay around the loading box should allow closing the fragment.

This contradicts with requirements from Slack

```
Just to clarify  expected results:
1. user should not be able to close the hcatcha by clicking on loading logo
2. SDK must hide loading logo, only which hcapthca fully loaded and visible to user
...

[Answers]
1. yes, it should decrease the number of accidental taps (the overlay is expected to be still interactive)
2. ideally, there shouldn’t be any delay between the loading screen disappearing and when the challenge is shown
```

@DSergiu correct me if I got this in a wrong way